app/code/Magento/Sales/etc/adminhtml/system.xml

```diff
@@ -49,9 +49,9 @@
                 </field>
             </group>
             <group id="zerograndtotal_creditmemo" translate="label" type="text" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="1">
-                <label>Allow Zero GrandTotal</label>
+                <label>Allow Zero GrandTotal for Credit Memos</label>
                 <field id="allow_zero_grandtotal" translate="label" type="select" sortOrder="1" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
-                    <label>Allow Zero GrandTotal for Creditmemo</label>
+                    <label>Allow Zero GrandTotal for Credit Memos</label>
```

Comment:
There are hundreds of "reditmemo" occurrences in code, including `csv` files. Please revert "credtimemo" -> "credit memo" changes.